.circleci/e2e-job-ignore-patterns.txt

```diff
@@ -6,5 +6,15 @@
 .log
 .pdf
 .gitignore
-.java
-.kt
\ No newline at end of file
+Test.java
+.kt
+.tsx
```

Comment:
This would basically ignore most UI code changes

Reply:
hmm, I want to ignore test files, not ignore src code changes. what can I change to?

Reply:
Not sure what you mean be test files. Are you talking about unit test files in the UI codebase? .spec.tsx should match those

Reply:
you're correct. I updated patterns for ui and api tests to `Test.kt` and
`.spec.tsx`